### DISCUSSION TITLE
如何获取消息

### DISCUSSION THREAD

**Discussion by police0909** [Mrs4s/go-cqhttp#2006] [Q&A]
不是很清楚这个api是怎么用的，怎么获取别人发来的信息呢？

http://127.0.0.1:5700/get_msg?message_id=-718467894

这个API，好像是通过message_id来获取信息详情。
那别人刚发来的信息，我怎么知道这个message_id呢？

**Comment by ilihu** [comment]
开启ws，或者http监听gocq推送过来的消息（gocq同时支持正反向ws，和http），这个消息里就有message_id
